### PR TITLE
sc-2189 verification RPC test

### PR DESCRIPTION
**Pull request by pdeziel**
This adds a test for the GDS Verification RPC.

### CODE REVIEW

File: pkg/gds/gds_test.go [hunk 1]
```diff
@@ -105,6 +106,59 @@ func (s *gdsTestSuite) TestRegister() {
...
+	s.LoadFullFixtures()
+	require := s.Require()
+	ctx := context.Background()
+
+	id := "d9da630e-41aa-11ec-9d29-acde48001122"
+
+	// The reference fixture doesn't contain the updated timestamp, so we retrieve the
+	// real VASP object here for comparison purposes.
+	vasp, err := s.svc.GetStore().RetrieveVASP(id)
+	require.NoError(err)
```
Comment (pdeziel) [inline pkg/gds/gds_test.go:121]: The problem here is that when `generateDB` creates the VASPs in the store the `lastUpdated` timestamp gets updated, meaning the field on the VASP is different than the field in the reference fixture.

File: pkg/gds/gds.go [hunk 2]
```diff
@@ -347,8 +347,8 @@ func (s *GDS) Search(ctx context.Context, in *api.SearchRequest) (out *api.Searc
 	return out, nil
 }
 
-// Status returns the status of a VASP including its verification and service status if
-// the directory service is performing health check monitoring.
+// Verification returns the status of a VASP including its verification and service
```
Comment (rebeccabilbro) [inline pkg/gds/gds.go:350]: Thank you for catching this docstring error!

File: pkg/gds/gds.go [hunk 3]
```diff
@@ -372,7 +372,7 @@ func (s *GDS) Verification(ctx context.Context, in *api.VerificationRequest) (ou
 
 		vasp = vasps[0]
 	default:
-		log.Warn().Str("rpc", "lookup").Msg("no arguments supplied")
+		log.Warn().Str("rpc", "verification").Msg("no arguments supplied")
```
Comment (rebeccabilbro) [inline pkg/gds/gds.go:375]: Very glad that you're doing these tests 🙂  good copypasta 🍝  catch!

**Review by rebeccabilbro — APPROVED** [review]
Very good PR @pdeziel, nice catches with those copy and paste errors!

**Pull request merged**